.travis.yml

```diff
@@ -0,0 +1,59 @@
+language: php
+sudo: required
```

Comment:
Why is that? I don't really see a reason. It works fine like this, the documentation states it must be first level, not the first line in the file.